zerver/context_processors.py

```diff
@@ -56,19 +56,19 @@ def zulip_default_context(request):
     """
     realm = get_realm_from_request(request)
 
+    realm_uri = settings.SERVER_URI
+    realm_name = None
+    realm_icon = None
+    realm_description = None
+    realm_invite_required = False
```

Comment:
Yeah, I think in this case, with a batch of several, I definitely prefer the two parallel cases.  Swapping the order seems fine.